codecov.yml

```diff
@@ -12,7 +12,3 @@ coverage:
       default:
         threshold: 0.25%
     patch: off
-
-ignore:
-  # This package is not shipped.
-  - "Tests"
```

Comment:
It's a little silly to require 100% coverage in unit tests but given the complexity of this library I see some benefit.

We can always re-add this `ignore` if maintaining this level of coverage proves problematic. But I like striving for full coverage in a library like this.